### ISSUE TITLE
The content of the file is newer when on remote container

### ISSUE THREAD

**Issue by dionysiosarvanitis** [microsoft/vscode#89155]
- VSCode Version: 1.41.1
- OS Version: Windows 10 1909 18363.593

Steps to Reproduce:

1. Open a file in remote container
2. Edit and save it multiple times quickly

Tried manually disable all extension except for remote. No formatters at all. Date inside container is synced with host.

Message I'm getting below:

<details><summary>Dev tools trace</summary>
Failed to save 'webpack.mix.js': The content of the file is newer. Please compare your version with the file contents.
onDidNotificationChange @ notificationsAlerts.ts:40
(anonymous) @ notificationsAlerts.ts:26
fire @ event.ts:581
addNotification @ notifications.ts:171
notify @ notificationService.ts:101
onSaveError @ textFileSaveErrorHandler.ts:179
onSaveError @ textFileEditorModel.ts:932
(anonymous) @ textFileEditorModel.ts:792
processTicksAndRejections @ internal/process/task_queues.js:89
Promise.then (async)
(anonymous) @ textFileEditorModel.ts:748
processTicksAndRejections @ internal/process/task_queues.js:89
Promise.then (async)
doSave @ textFileEditorModel.ts:700
save @ textFileEditorModel.ts:626
(anonymous) @ textFileService.ts:659
doSaveAllFiles @ textFileService.ts:658
doSaveAll @ textFileService.ts:539
saveAll @ textFileService.ts:533
save @ textFileService.ts:507
save @ editor.ts:566
(anonymous) @ editorService.ts:696
save @ editorService.ts:688
K @ fileCommands.ts:387
V @ fileCommands.ts:349
handler @ fileCommands.ts:399
invokeFunction @ instantiationService.ts:67
_tryExecuteCommand @ commandService.ts:84
(anonymous) @ commandService.ts:73
Promise.then (async)
executeCommand @ commandService.ts:73
_doDispatch @ abstractKeybindingService.ts:196
_dispatch @ abstractKeybindingService.ts:155
(anonymous) @ keybindingService.ts:218
</details>

**Comment by bpasero** [comment]
@dionysiosarvanitis when you compare the file then do you see any differences?

As a workaround you can download insiders [1] and configure `files.preventSaveConflicts: false` as a workspace setting. 

[1] https://code.visualstudio.com/insiders/

**Comment by paulriley** [comment]
I have seen the same problem in two locations since Docker for Windows updated to 2.2, developing .NET Core apps.

It happens on any attempt to save (autoSave or Ctrl-S).

If you ask it to compare, it shows all changes since the file was opened. If you then tell it to keep your changes, it does so, but that's a big change to workflow.

If you try to build using the terminal then it tries to build using the saved file (before any changes you've made). But, if you reload VS Code or even rebuild the container, the same changes remain in memory somewhere.

**Comment by dionysiosarvanitis** [comment]
Yes, If you compare, you can see some diffs. Thankfully at my home PC it works fine. The only difference between the two systems is the OS version: Windows_NT x64 10.0.18363 vs 10.0.19041. The latest version is the one without any issues. Thanks for the workaround.

**Comment by bpasero** [comment]
Is it possible that some other process is modifying the file? Are the changes show a version that was previously saved by you?

**Comment by dionysiosarvanitis** [comment]
Yes, it shows diffs previously saved by me. I tried with AV disabled. Can't think of anything else.

**Comment by bpasero** [comment]
@dionysiosarvanitis and you say this happened ever since Docker updated, so not something VSCode broke in recent updates?

**Comment by dionysiosarvanitis** [comment]
No, paulriley said that! I'm not sure since when it started, but I guess it was after windows upgraded to 1909 from 1809.
